patches/@kayahr+jest-electron-runner+29.14.0.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/node_modules/@kayahr/jest-electron-runner/lib/main/electron/TestRunner.js b/node_modules/@kayahr/jest-electron-runner/lib/main/electron/TestRunner.js
+index 4485862..12e0042 100644
+--- a/node_modules/@kayahr/jest-electron-runner/lib/main/electron/TestRunner.js
```

Comment:
I wasn't experiencing any issues with the test runner.
Could try updating to latest package first maybe?

Reply:
I've alreafy tried reinstalling dependendcies by removing node_modules and it didnt helped. Could you please send your yarn.lock?